src/providers/DevicesProvider/docs/VideoInputProvider.stories.mdx

```diff
@@ -2,19 +2,20 @@
 
 # VideoInputProvider
 
-Provides a list of the user's available `video-input` devices, the currently selected device.
+Provides a list of the user's available video input devices and the currently selected video input device.
 
 ## State
 
 ```javascript
```

Comment:
Would appreciate updating this based on AudioInputProvider comments.

Reply:
Updated.